docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -38,9 +43,10 @@ Powder
 .. image:: ../images/s_powder_scheme.png
     :align: center
 
-From DFT calculations one usually obtains phonon data for infinite single crystal (there are ways to perform
-calculations for molecule, e.g, gas phase). In order to compare it with experiment in which sample is in the form of
-powder one has to perform powder averaging. Usually semi-empirical formula is used [2]_, [3]_:
+In DFT studies of solid materials, one usually applies periodic boundary conditions representing an infinite single crystal.
```

Comment:
```suggestion
In DFT studies of solid materials, one usually applies periodic boundary conditions to represent an infinite single crystal.
```

Reply:
Not quite the same implication, but it does read better. (We don't always choose PBC because we _want_ an infinite crystal, but using PBC means we always have one.)

Reply:
yes, I missed that. Just need to tighten up the language.